packages/dev-utils/README.md

```diff
@@ -0,0 +1,22 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Sentry JavaScript SDK Developer Utilities
+
+[![npm version](https://img.shields.io/npm/v/@sentry-internal/dev-utils.svg)](https://www.npmjs.com/package/@sentry-internal/dev-utils)
+[![npm dm](https://img.shields.io/npm/dm/@sentry-internal/dev-utils.svg)](https://www.npmjs.com/package/@sentry-internal/dev-utils)
+[![npm dt](https://img.shields.io/npm/dt/@sentry-internal/dev-utils.svg)](https://www.npmjs.com/package/@sentry-internal/dev-utils)
+[![typedoc](https://img.shields.io/badge/docs-typedoc-blue.svg)](http://getsentry.github.io/sentry-javascript/)
+
+## Links
+
+- [Contribution Guide](https://github.com/getsentry/sentry-javascript/blob/master/CONTRIBUTING.md)
+- [SDK Developer Docs](https://develop.sentry.dev/sdk/)
+
+## General
+
+Common utilities used during development of the Sentry JavaScript SDKs.
```

Comment:
Should we add a note here that this is an internal package which people shouldn't use and not expect to follow semver (similarily to utils)?

Reply:
Can do. It's in the `@sentry-internal` scope, so I figured it was implied, but I can definitely mention it here, too.